packages/contracts/contracts/libraries/rlp/Lib_RLPWriter.sol

```diff
@@ -228,11 +227,13 @@ library Lib_RLPWriter {
             src += 32;
         }
 
-        uint256 mask = 256 ** (32 - len) - 1;
-        assembly {
-            let srcpart := and(mload(src), not(mask))
-            let destpart := and(mload(dest), mask)
-            mstore(dest, or(destpart, srcpart))
+        unchecked {
```

Comment:
mirroring kelvin's nit above: could move the `assembly` block outside the `unchecked`

Reply:
To be consistent I have updated both instances of the `unchecked` math to be standalone rather than push logic into them, i.e. 
```
uint256 mask;
        unchecked {
            mask = 256 ** (32 - (_length % 32)) - 1;
        }

        assembly {
            mstore(
                dest,
                or(
                    and(mload(src), not(mask)),
                    and(mload(dest), mask)
                )
            )
        }
        return out;
```
instead of

```
unchecked {
            uint256 mask = 256 ** (32 - (_length % 32)) - 1;
            assembly {
                mstore(
                    dest,
                    or(
                        and(mload(src), not(mask)),
                        and(mload(dest), mask)
                    )
                )
            }
            return out;
        }